package.json

```diff
@@ -24,5 +24,8 @@
     "mocha": "^9.2.2",
     "shx": "^0.3.4",
     "typescript": "^4.7.4"
+  },
+  "dependencies": {
+    "yakumo": "^0.3.6"
```

Comment:
What is diff here for?

Reply:
for the vscode's dependency completion😂😂

Reply:
> What is diff here for?

 Can this pull request be merged?

Reply:
It's generally ok, but I need to make some adjustments before merging.

Reply:
OK,I got it